cheatsheets/gleam-for-elm-users.md

```diff
@@ -0,0 +1,699 @@
+---
+layout: page
+title: Gleam for Elm users
+---
+
+- [Overview](#overview)
+- [Comments](#comments)
+- [Variables](#variables)
+  - [Variables type annotations](#variables-type-annotations)
+- [Constants](#constants)
+- [Functions](#functions)
+  - [Function type annotations](#function-type-annotations)
+  - [Function heads](#function-heads)
+  - [Function overloading](#function-overloading)
+  - [Referencing functions](#referencing-function)
+  - [Calling anonymous functions](#calling-anonymous-functions)
+  - [Labelled arguments](#labelled-arguments)
+- [Modules](#modules)
+  - [Exports](#exports)
+- [Blocks](#blocks)
+- [Data types](#data-types)
+  - [Numbers](#numbers)
+  - [Strings](#strings)
+  - [Tuples](#tuples)
+  - [Records](#records) TODO
+  - [Lists](#lists)
+  - [Dicts](#dicts)
+- [Operators](#operators)
+- [Type aliases](#type-aliases)
+- [Custom types](#custom-types) TODO
+  - [Maybe](#maybe)
+  - [Result](#result)
+- [Commands](#commands) TODO
+- [Architecture](#architecture) TODO
+- [Talking to other languages](#talking-to-other-languages) TODO
+- [Package management](#package-management)
+- [Implementation](#implementation)
+- [Other concepts](#other-concepts)
+  - [Atoms](#atoms)
+
+## Overview
+
+Elm and Gleam have similar goals of providing a robust and sound type system with an approachable
+set of features. Gleam started off with a similar syntax to Elm but as moved to use one that is more
+familiar to many developers.
```

Comment:
```suggestion
set of features.
```

An interesting bit of history, but not something I think belongs here. I wouldn't want any newcomer Elm syntax fans to feel let down by this change.